tests/unit_tests/bittensor_tests/utils/test_utils.py

```diff
@@ -168,7 +168,7 @@ def test_solve_for_difficulty_fast_registered_already():
         block_hash = '0xba7ea4eb0b16dee271dbef5911838c3f359fcf598c74da65a54b919b68b67279'
         subtensor = MagicMock()
         subtensor.get_current_block = MagicMock( return_value=1 )
-        subtensor.difficulty = 2560000000 # set high to make solving take a long time
+        subtensor.difficulty = 100000000000000000000000000 # set high to make solving take a long time
```

Comment:
I probably missed something here, why is difficulty changed so high?

Reply:
Cause it was supposed to be a number that is too big to get solved so that the pow can never complete and the registration can only get finished because the wallet got registered somewhere.